server/conf/evolutions/default/70.sql

```diff
@@ -0,0 +1,23 @@
+# --- !Ups
```

Comment:
**suggestion:** Could you add a comment to the top of this file explaining the change briefly. Something like "Copies over statuses from the programs table to the new application_statuses table"